### PR TITLE
Support email field on Location

### PR DESCRIPTION
**Pull request by tamcgoey**
As more international carriers are supported with `ActiveShipping`, we're needing to support more contact methods outside of just the current implemented fields: `phone` & `fax`.

This PR adds support for a `email` field on `Location`, this way email addresses can be passed on to a carrier.

### CODE REVIEW

**Comment by tamcgoey** [comment]
CI is failing do to a flaky UPS remote test. Should be save to deploy still.

File: lib/active_shipping/location.rb [hunk 1]
```diff
@@ -13,6 +13,7 @@ class Location
       address3: [:address3],
       phone: [:phone, :phone_number],
       fax: [:fax, :fax_number],
+      email: [:email, :email_address],
```
Comment (garethson) [inline lib/active_shipping/location.rb]: Do you really need `email_address` here? These attribute aliases lead to code that's hard to determine what's actually being passed in / used.

Comment (tamcgoey) [inline lib/active_shipping/location.rb]: Thought it would be useful in the library, but I think it's okay to drop it in this case.

**Comment by tamcgoey** [comment]
@garethson dropped the alias for `email`.

**Pull request merged**